llvm/lib/Target/RISCV/RISCVInstrInfoXSf.td

```diff
@@ -307,44 +307,68 @@ multiclass VPseudoVC_X<LMULInfo m, DAGOperand RS1Class,
                        Operand OpClass = payload2> {
   let VLMul = m.value in {
     let Defs = [VCIX_STATE], Uses = [VCIX_STATE] in {
-      def "PseudoVC_" # NAME # "_SE_" # m.MX : VPseudoVC_X<OpClass, RS1Class>;
-      def "PseudoVC_V_" # NAME # "_SE_" # m.MX : VPseudoVC_V_X<OpClass, m.vrclass, RS1Class>;
+      def "PseudoVC_" # NAME # "_SE_" # m.MX
+        : VPseudoVC_X<OpClass, RS1Class>,
+          Sched<[!cast<SchedWrite>("WriteVC_" # NAME # "_" # m.MX)]>;
```

Comment:
Is it the case that these instructions do not read any data? This only models writes.

Reply:
No, that's probably just me not understanding LLVM's scheduling well enough... Most of the VCIX instructions do indeed read data (mostly vector registers, but in a few cases also scalar ones). I guess that would need the `LMULSchedReads` (similar to the `LMULSchedWrites`)? (and then `LMULReadAdvance`?)

Reply:
Ok, I finally took another look at this. Right now I don't think my team would need `SchedRead`/`ReadAdvance`s for any of these instructions. But I'm happy to add them if you think that's important. So let me know what you think! (also, would you be ok to add it in a separate PR?) Thanks! 🙂

Reply:
As written, this code specifies scheduling information such that each VCIX instruction writes data but does not read data. That means that if a VCIX instruction reads data in reality (which it likely does, but correct me if I am wrong), then the scheduler will not check to see if the data it reads is ready based on the write latency entry.

For example:
```
a = op b
c = vcix a
```

If there is a SchedRead for `vcix` then the scheduler will check to see what the write latency on `a` was and use that information in determining when `vcix` should be scheduled.

Reply:
Oops, @topperc pointed out I am wrong:

> SchedRead is only used to lookup if the latency should be adjusted using a ReadAdvance. The dependency graph is defined by the register operands is tells the scheduler which instructions an instruction reads from

As a result, I think this PR is okay without having SchedRead. We can add in a follow up PR if it becomes important.

Reply:
Sounds good, thanks! 😄